tests/llmobs/_utils.py

```diff
@@ -147,7 +147,7 @@ def _llmobs_base_span_event(
     span_event = {
         "span_id": str(span.span_id),
         "trace_id": "{:x}".format(span.trace_id),
-        "parent_id": "undefined",
+        "parent_id": str(span.parent_id or "undefined"),
```

Comment:
This will break tests that have trace structures like `LLMObs Span --> APM Span --> LLMObs Span` since we propagate parent IDs from the nearest LLMObs span. For some reason these tests are being skipped on the llmobs/openai/botocore test suites, I would try running those tests locally to see what's breaking.

One quick way to fix the breaking tests is to introduce a utility function here, like so:
```python
from ddtrace.ext import SpanTypes

def _get_llmobs_parent_id(span):
    if not span._parent:
        return "undefined"
    parent = span._parent
    while parent is not None:
        if parent.span_type == SpanTypes.LLM:
            return str(parent.span_id)
        parent = parent._parent
```